scripts/gendocs/generate

```diff
@@ -142,23 +144,34 @@ $output
 EOF
 }
 
+# The branches named here exist in the `jetstack/cert-manager` repo.
+
+genversionwithcli "release-1.8" "next-docs"
+genversionwithcli "release-1.7" "docs"
+genversionwithcli "release-1.7" "v1.7-docs"
```

Comment:
The 1.7 and 1.8 API docs are now generated using the new wrapper function `genversionwithcli` which generates both API docs and CLI docs for a particular branch.